src/search/search.component.html

```diff
@@ -14,7 +14,7 @@
 		'bx--toolbar-search-container-expandable': tableSearch && expandable,
 		'bx--toolbar-search-container-active': tableSearch && expandable && active
 	}"
-	role="search"
+	role="searchbox"
```

Comment:
Seems like the accessibility error is caused by `tabindex`, not the role.

```suggestion
	role="search"
```

![image](https://user-images.githubusercontent.com/38994122/189411759-29a70827-1f62-416f-b203-edae36407629.png)